packages/excalidraw/clipboard.ts

```diff
@@ -270,6 +255,34 @@ export const readSystemClipboard = async () => {
     throw error;
   }
 
+  try {
+    if (navigator.clipboard?.read) {
+      const readClipboard = await navigator.clipboard?.read();
+      const textItem = readClipboard.find((item) => 
+        item.types.includes("text/plain")
+      );
+      if (textItem) {
+        const readText = await navigator.clipboard.readText();
+        if (readText) {
+          return { "text/plain": readText };
```

Comment:
This seems a bit strange, as it assumes all clipboard items are of the same exact type. Could we extend it to support pasting any types together? At the same time, when one type fails, the whole paste should not fail and continue.

Reply:
yeah, I think this is some legacy code. I've now put it as a fallback to `read()` instead.